features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/admin-dashboard/site/blocks/theme-list/block.jag

```diff
@@ -0,0 +1,44 @@
+<%
+include("/jagg/jagg.jag");
+var site = require("/site/conf/site.json");
+jagg.block("theme-list", {
+    initializer: function (data) {
+
+    },
+
+    getInputs: function () {
```

Comment:
change `getInputs` function body to `function () { return {}; }` to be more descriptive